src/main/java/org/jenkinsci/plugins/GithubAuthorizationStrategy.java

```diff
@@ -84,11 +90,20 @@ public ACL getRootACL() {
     }
 
     @Nonnull
+    public ACL getACL(@Nonnull AbstractItem item) {
+        if(item instanceof WorkflowMultiBranchProject) {
```

Comment:
Updated to use MultiBranch: https://github.com/jenkinsci/github-oauth-plugin/pull/78/commits/15233fb3db7ccaea0b845fadca192581e52f95a4